README.md

```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
-| Multiple versions            |           | √         | √           |
-| Multiple APIs                |           |           | √           |
-| API catalog                  |           |           | √           |
-|                              |           |           |             |
-| **Additional features**      |           |           |             |
-| Try-it console               |           | √         | √           |
-| Automated code samples       |           | √         | √           |
-| Deep links                   |           | √         | √           |
-| More SEO control             |           |           | √           |
-| Contextual docs              |           |           | √           |
-| Landing pages                |           |           | √           |
-| React hooks for more control |           |           | √           |
-| Personalization              |           |           | √           |
-| Analytics integrations       |           |           | √           |
-| Feedback                     |           |           | Coming Soon |
-
-Refer to the Redocly's documentation for more information on these products:
-
-- [Portals](https://redocly.com/docs/developer-portal/introduction/)
-- [Reference](https://redocly.com/docs/api-reference-docs/getting-started/)
-- [Redoc](https://redocly.com/docs/redoc/quickstart/intro/)
-
-## Features
 - Responsive three-panel design with menu/scrolling synchronization
-- [Multiple deployment options](https://redocly.com/docs/redoc/quickstart/intro/)
-- [Server-side rendering (SSR) ready](https://redocly.com/docs/redoc/quickstart/cli/#redoc-cli-commands)
+- Support for OpenAPI 3.1, OpenAPI 3.0 and Swagger 2.0
+- [Multiple deployment options](https://redocly.com/docs/redoc/)
+- High-level grouping in side menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
 - Ability to integrate your API introduction into the side menu
 - [Simple integration with `create-react-app`](https://redocly.com/docs/redoc/quickstart/react/)
-
-  [Example repo](https://github.com/APIs-guru/create-react-app-redoc)
-- [Command-line interface to bundle your docs into a **zero-dependency** HTML file](https://redocly.com/docs/cli/commands/build-docs/)
-- Neat **interactive** documentation for nested objects <br>
-  ![](docs/images/nested-demo.gif)
-
-## Customization options
-[<img alt="Customization services" src="http://i.imgur.com/c4sUF7M.png" height="60px">](https://redocly.com/#services)
-- High-level grouping in side-menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
-- Branding/customizations using the [`theme` option](https://redocly.com/docs/api-reference-docs/configuration/theming/)
-
-## Support
-- OpenAPI v3.0 support
-- Basic OpenAPI v3.1 support
-- Broad OpenAPI v2.0 feature support (yes, it supports even `discriminator`) <br>
```

Comment:
I would move this gif into the features somehow.

Reply:
No more gifs! Especially not related to OpenAPI 2.0.